doc/building.html

```diff
@@ -614,10 +614,8 @@ <h3 id="clang">clang</h3>
 <code>--with-toolchain-type=clang</code>.</p>
 <h3 id="apple-xcode">Apple Xcode</h3>
 <p>The oldest supported version of Xcode is 13.0.</p>
-<p>You will need the Xcode command line developer tools to be able to
-build the JDK. (Actually, <em>only</em> the command line tools are
-needed, not the IDE.) The simplest way to install these is to run:</p>
-<pre><code>xcode-select --install</code></pre>
+<p>You will need to download Xcode either from the App Store or specific
+versions can be easily located via <a href="https://xcodereleases.com">https://xcodereleases.com</a>.</p>
```

Comment:
If I run `make update-build-docs`, that does not turn into a link tag. Did you hand edit the html or convert it automatically?

```suggestion
versions can be easily located via https://xcodereleases.com.</p>
```

Reply:
sorry I manually make it a link tag, that's incorrect, I'll fix it now.

Reply:
You can make it a link using markdown []() syntax.

Reply:
> You can make it a link using markdown `[]()` syntax.

yeah I'll do that

Reply:
I will approve once this is fixed, looks good

Reply:
updated PTAL